src/components/RulesModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+      <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+          <div class="modal-header">  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+            <h3 class="modal-title">Programming Wars Rules</h3>
+          </div>
+          <div class="modal-body" style="text-align: left">
+            <h4>Introduction</h4>
+            <p>The basic elements of a program are simple to understand, but can be used to create complex
+              computer programs. In this fun, fast-paced game you try to complete a simple program before
+              the opponent player or team.</p>
```

Comment:
"Team" seems confusing as the "player" only sees other "players". I may have written "team" in the original documentation because with real cards you can see that you are playing with a team (and teams were used due to the limited number of cards, which is no longer an issue.)